src/main/scala/mesosphere/marathon/storage/repository/GcActor.scala

```diff
@@ -79,9 +86,24 @@ private[storage] class GcActor[K, C, S](
   private var lastCompactStart = Instant.now()
   private val compactTime = Kamon.metrics.histogram("GarbageCollector.compactTime", Time.Milliseconds)
 
-  startWith(Idle, IdleData)
+  if (cleaningInteveral <= 0.millis) {
+    startWith(ReadyForGc, EmptyData)
+  } else {
+    startWith(Resting, EmptyData)
+  }
 
-  when(Idle) {
+  when(Resting) {
+    case Event(WakeUp, _) =>
+      goto(ReadyForGc) using EmptyData
+    case Event(StoreEntity(promise), _) =>
```

Comment:
What kind of event is this?

Reply:
@jeschkies it's an event that sent by DeploymentRepository when an app/pod/root is stored in the corresponding repository. 

@alenkacz I don't think it's possible to ignore them since integration tests breake without it